### DISCUSSION TITLE
speed tests for the curve x25519_kyber768

### DISCUSSION THREAD

**Discussion by vkosuri** [open-quantum-safe/liboqs#1672] [Q&A]
Do we have any speed tests for the curve x25519_kyber768, I have seen the speed tests for KEM operations not for the curve x25519_kyber768.

Thanks for all your help.

**Comment by dstebila** [comment]
No, because we don't have elliptic curve algorithms (x25519) in liboqs directly.

**Reply by dstebila** [reply]
You can see some results with elliptic curves in the 5th and 6th link on https://openquantumsafe.org/benchmarking/ which does speed testing in OpenSSL of some hybrid combinations.
